### PR TITLE
feat(settings/native/android): Fixed scroll inside conference settings screen

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

**Review by gabiborlea — APPROVED** [review]

File: react/features/mobile/navigation/screenOptions.ts [hunk 1]
```diff
@@ -244,6 +244,7 @@ export const lobbyNavigationContainerScreenOptions = {
  * Screen options for settings navigation container screen.
  */
 export const settingsNavigationContainerScreenOptions = {
+    ...modalPresentation,
```
Comment (saghul) [inline react/features/mobile/navigation/screenOptions.ts:247]: We don't present it modally on the welcome page. Does this break anything there?

Should we have 2 screens wrapping the same component with different settings?

Comment (Calinteodor) [reply react/features/mobile/navigation/screenOptions.ts:247]: These screen options are applied only on the ConferenceNavigationContainer.

Comment (gabiborlea) [reply react/features/mobile/navigation/screenOptions.ts:247]: https://github.com/jitsi/jitsi-meet/blob/4cc4c25691c4275919aed5cb23110ef4720519e0/react/features/mobile/navigation/components/settings/components/SettingsNavigationContainer.tsx#L38

Comment (saghul) [reply react/features/mobile/navigation/screenOptions.ts:247]: Ah kewl!

**Pull request merged**